src/utilities/api-requests.ts

```diff
@@ -40,35 +45,37 @@ function parseRetryHeaderToMs(response: Response, defaultDelay: number): number
     return Math.max(0, waitTime);
 }
 
-async function attemptRequest<T>(uri: string, defaultValue: T, attemptNumber: number): Promise<T> {
-    const response = await fetch(uri);
+async function attemptRequest<T>(uri: string, attemptNumber: number): Promise<T> {
+    const response = await fetch(uri).catch((reason: Error) => {return reason.message});
```

Comment:
Codacy found an issue: [This arrow function body can be simplified by omitting the curly braces and the keyword 'return'.](https://app.codacy.com/gh/Project-Lavinia/Lavinia-client/pullRequest?prid=5985654)